packages/sources/universal-v3/src/endpoint/lambda.ts

```diff
@@ -0,0 +1,203 @@
+import type {
+  Transport,
+  TransportGenerics,
+  TransportDependencies,
+} from '@chainlink/external-adapter-framework/transports'
+import type {
+  AdapterRequest,
+  AdapterResponse,
+  RequestGenerics,
+} from '@chainlink/external-adapter-framework/util'
+import type { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+import type { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import type { ResponseCache } from '@chainlink/external-adapter-framework/cache/response'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { LambdaController } from '../utils/LambdaController'
+import { customSettings } from '../index'
+import { decrypt } from '../utils/secretsDecrypter'
+import type { SandboxOutput, Response } from '../utils/buildResponse'
+import { ResponseBuilder } from '../utils/buildResponse'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+
+export const inputParameters: InputParameters = {
+  source: {
+    description: 'JavaScript source code to be executed',
+    type: 'string',
+    required: true,
+  },
+  subscriptionId: {
+    description: 'Subscription Id used by the request',
+    required: true,
+  },
+  requestId: {
+    description: 'Request Id unique to each request',
+    required: true,
+  },
+  maxResponseBytes: {
+    description: 'Maximum size of response data, in bytes',
+  },
+  numAllowedQueries: {
```

Comment:
Where does `numAllowedQueries` and `maxResponseBytes` come from? They aren't in the request CBOR - so they are set staticly in the OCR plugin?

Reply:
They are optional params that we can add in the future if we want to charge different amounts based on the size of the response or the number of queries.  They are fully functional right now, but not required params.  They have default values within the Lambda sandbox (256 byte max response & 5 max queries, both of which are configurable via env var)